content/blog/2021/06/kwctl-intro-for-kubernetes-administrators.md

```diff
@@ -0,0 +1,307 @@
+---
+title: Introducing kwctl to Kubernetes Administrators
+authors:
+    - Flavio Castelli
+date: 2021-06-08
+---
+
+We are pleased to announce the availability of a new tool within the Kubewarden
+project: [kwctl](https://github.com/kubewarden/kwctl).
+
+kwctl is a command line utility designed to help both policy authors
+and Kubernetes administrators.
+
+This blog post focuses on the user experience of Kubernetes administrators,
+future ones will cover the policy developer side of the story.
+
+## A real world example: controlling container capabilities
+
+The main character of today's story is Alice. Alice is a Kubernetes
+administrator who wants to keep her Kubernetes cluster secure.
+
+Alice uses many solutions to accomplish that, including
+[Kubernetes Pod Security Policies](https://kubernetes.io/docs/concepts/policy/pod-security-policy/).
+
+Pod Security Policies are currently deprecated and are going to be removed from Kubernetes 1.25.
+Due to that, Alice wants to find an alternative to Kubernetes PSP.
```

Comment:
So that you can use "PSP" here ;-)

Reply:
Maybe "PSPs" is better in this context ?

Reply:
good points